Web/starter_project_iconics/components/Homepage/Landing.tsx

```diff
@@ -0,0 +1,38 @@
+import Image from 'next/image'
+import { BsArrowRightShort } from 'react-icons/bs'
+
+const Landing = () => {
+  return (
+    <div className="flex flex-row justify-between p-16">
+      <div className="flex flex-col items-center justify-between basis-5/12">
+        <div className="flex flex-col items-center justify-between">
+          <h1 className="text-7xl font-extrabold">
+            Africa to <span className="text-[#264FAD]">Silicon Valley</span>
```

Comment:
let's define colors globally and reuse then in different places (this coould be the accent color for instance idk)

Reply:
yeah, that's how it's going to be done, a person has been assigned and when they finish, I will change it to be that way.